src/component/utils/exploration/__tests__/__snapshots__/wrapBlockNodes.js.snap

```diff
@@ -0,0 +1,595 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders block with child that have wrapperElement 1`] = `
+<blockquote
+  data-block={true}
+  data-editor="editor"
+  data-offset-key="A-0-0"
+>
+  <div
+    className="public__DraftStyleDefault__block public__DraftStyleDefault__ltr"
+    data-offset-key="A-0-0"
+  >
+    <ol
+      className="public__DraftStyleDefault__ol"
+      data-offset-key="B-0-0"
+    >
+      <li
+        data-block={true}
+        data-editor="editor"
+        data-offset-key="B-0-0"
+      >
+        <div
+          className="public__DraftStyleDefault__block public__DraftStyleDefault__ltr"
+          data-offset-key="B-0-0"
+        >
+          <span
+            data-offset-key="B-0-0"
+            style={Object {}}
+          >
+            <span
+              data-text="true"
+            >
+              first list item
+            </span>
+          </span>
+        </div>
+      </li>
+    </ol>
+  </div>
+</blockquote>
+`;
+
+exports[`renders block with children that have same wrapperElement 1`] = `
+<blockquote
+  data-block={true}
+  data-editor="editor"
+  data-offset-key="A-0-0"
+>
+  <div
+    className="public__DraftStyleDefault__block public__DraftStyleDefault__ltr"
+    data-offset-key="A-0-0"
+  >
+    <ol
+      className="public__DraftStyleDefault__ol"
+      data-offset-key="B-0-0"
+    >
+      <li
+        data-block={true}
+        data-editor="editor"
+        data-offset-key="B-0-0"
+      >
+        <div
+          className="public__DraftStyleDefault__block public__DraftStyleDefault__ltr"
+          data-offset-key="B-0-0"
+        >
+          <span
+            data-offset-key="B-0-0"
+            style={Object {}}
+          >
+            <span
+              data-text="true"
+            >
+              first list item
+            </span>
+          </span>
+        </div>
+      </li>
+      <li
+        data-block={true}
+        data-editor="editor"
+        data-offset-key="C-0-0"
+      >
+        <div
+          className="public__DraftStyleDefault__block public__DraftStyleDefault__ltr"
+          data-offset-key="C-0-0"
+        >
+          <span
+            data-offset-key="C-0-0"
+            style={Object {}}
+          >
+            <span
+              data-text="true"
+            >
+              second list item
+            </span>
+          </span>
+        </div>
+      </li>
+    </ol>
+  </div>
+</blockquote>
+`;
+
+exports[`renders block with nested child that have different wrapperElement 1`] = `
+<blockquote
```

Comment:
This output only seems to show 
```
<blockquote (A)> 
  <ol (c)>
  </ol>
</blockquote>
```

What about 'B', the `ul`? I would expect it to look like this:

```
<blockquote (A)> 
  <ul (B)>
    <ol (C)>
    </ol>
  </ul>
</blockquote>
```